doc/cephfs/administration.rst

```diff
@@ -92,6 +92,14 @@ The CephX IDs authorized to the old file system name need to be reauthorized
 to the new name. Any on-going operations of the clients using these IDs may be
 disrupted. Mirroring is expected to be disabled on the file system.
 
+::
+
+    fs swap <fs1-name> <fs2-name> [--swap-fscids] [--yes-i-really-mean-it]
```

Comment:
Isn't this `ceph fs swap ...`?

Reply:
We don't write in `ceph` in CephFS commands in this doc.

Reply:
really! wow.

Reply:
That's weird and should be fixed (not in this PR).